ci/release/template/man/d2.1

```diff
@@ -9,7 +9,7 @@
 .Op Fl -watch Ar false
```

Comment:
I ran ./install.sh after updating this but the man pages still rendered the old version. When I manually updated the man page though it showed me this:
![Screen Shot 2022-11-17 at 1 34 03 PM](https://user-images.githubusercontent.com/6413609/202571410-0c3ca76b-2a1d-47c1-8c23-17cf440798f2.png)

Reply:
just run `man ./path/to/man`